src/polycubed/src/base_cube.cpp

```diff
@@ -21,12 +21,24 @@
 namespace polycube {
 namespace polycubed {
 
+uint64_t genBaseTime() {
```

Comment:
I suggest to comment this code a little bit more.

Reply:
I agree, why are you using two different clocks?

Reply:
Comments + DOC added. I fixed it, the second timer was a mistake